packages/maskbook/src/plugins/Wallet/database/types.ts

```diff
@@ -23,6 +24,7 @@ export interface WalletRecord {
     erc20_token_whitelist: Set<string>
     /** A list of untrusted ERC20 token address */
     erc20_token_blacklist: Set<string>
+    provider: ProviderType
```

Comment:
There should be no `provider` property for WalletRecord anymore. If you wanna get the provider type of the currently selected wallet plz use `currentSelectedWalletProviderSettings` in `packages/maskbook/src/plugins/Wallet/settings.ts`.

Reply:
I Figure it out！